### ISSUE TITLE
Bad presentation of successful rollback

### ISSUE THREAD

**Issue by bittrance** [burtcorp/cuffsert#7]
The following sequence of events:
```
16:50:30 UTC+0100	DELETE_COMPLETE		AWS::CloudFormation::Stack	bittrance-test-emr-cluster	
16:50:29 UTC+0100	DELETE_COMPLETE		AWS::EC2::SecurityGroup		ClusterSg	
16:50:27 UTC+0100	DELETE_IN_PROGRESS	AWS::EC2::SecurityGroup		ClusterSg	
16:50:26 UTC+0100	DELETE_COMPLETE		AWS::EMR::Cluster		DruidIngestCluster	
16:50:18 UTC+0100	DELETE_IN_PROGRESS	AWS::CloudFormation::Stack	bittrance-test-emr-cluster	The following resource(s) failed to create: [DruidIngestCluster]. . Delete requested by user.
16:50:16 UTC+0100	CREATE_FAILED		AWS::EMR::Cluster		DruidIngestCluster		ServiceAccessSecurityGroup cannot be the same security group as EmrManagedMasterSecurityGroup or EmrManagedSlaveSecurityGroup
16:50:16 UTC+0100	CREATE_IN_PROGRESS	AWS::EMR::Cluster		DruidIngestCluster		
16:50:13 UTC+0100	CREATE_COMPLETE		AWS::EC2::SecurityGroup		ClusterSg	
16:50:12 UTC+0100	CREATE_IN_PROGRESS	AWS::EC2::SecurityGroup		ClusterSg	Resource creation Initiated
16:49:55 UTC+0100	CREATE_IN_PROGRESS	AWS::EC2::SecurityGroup		ClusterSg	
16:49:50 UTC+0100	CREATE_IN_PROGRESS	AWS::CloudFormation::Stack	bittrance-test-emr-cluster	User Initiated
```
Resulted in this output:
```
15:50:16+0000  DruidIngestCluster[AWS::EMR::Cluster] ServiceAccessSecurityGroup cannot be the same security group as EmrManagedMasterSecurityGroup or EmrManagedSlaveSecurityGroup
<green plus>,<white plus>,<white plus>
Done.
```

1. The stack marker should not be green; it ends up being deleted 
1. The end message should reflect that we rolled back, both text and color.

**Comment by bittrance** [comment]
Regarding 1., the following sequence:
```
17:18:16 UTC+0100	CREATE_COMPLETE	AWS::CloudFormation::Stack	bittrance-test-emr-cluster	
17:18:13 UTC+0100	CREATE_COMPLETE	AWS::EMR::Cluster	DruidIngestCluster	
17:12:37 UTC+0100	CREATE_IN_PROGRESS	AWS::EMR::Cluster	DruidIngestCluster	Resource creation Initiated
17:12:36 UTC+0100	CREATE_IN_PROGRESS	AWS::EMR::Cluster	DruidIngestCluster	
17:12:32 UTC+0100	CREATE_COMPLETE	AWS::EC2::SecurityGroup	ClusterSg	
17:12:31 UTC+0100	CREATE_IN_PROGRESS	AWS::EC2::SecurityGroup	ClusterSg	Resource creation Initiated
17:12:15 UTC+0100	CREATE_IN_PROGRESS	AWS::EC2::SecurityGroup	ClusterSg	
17:12:12 UTC+0100	CREATE_IN_PROGRESS	AWS::CloudFormation::Stack	bittrance-test-emr-cluster	User Initiated
```
resulted in `<green plus><white dot><green plus>`. Reasonably, this should be three green pluses.

**Comment by bittrance** [comment]
This is likely to be a combination of two things:
- using describe_stacks to check for completion (which risks missing the last batch of events)
- the describe_stack_events API which returns events in reverse chronological order

The former is addressed by 010e43b6d3600a0d133fdbf5d56a06af40dd0bfb. The latter remains to fix. Thus, this is probably the same problem as remains on #4 .
